fiftyone/server/main.py

```diff
@@ -597,6 +623,10 @@ async def on_distributions(self, group):
         else:
             results = []
 
+        for stage_dict in state.filters.values():
```

Comment:
maybe add a factory method to `DatasetView` for constructing a view from a list of stage dicts? this pattern appears multiple places in the code now, I believe

Reply:
Good point, improving this is in progress in order to best resolve #693